### DISCUSSION TITLE
Is test result shows that "RUNNING in chrome" is the expected behavious when testing electron app?

### DISCUSSION THREAD

**Discussion by codenoobforreal** [webdriverio/webdriverio#13982] [Q&A]
I created a project by following the "WebdriverIO Tutorials: Getting Started with Electron Testing" video.I found out the result show the "RUNNING in chrome" info.Is this expected behavious?
I had to do `import { test } from "mocha"` to make the tests pass otherwise it will fail cuz `ReferenceError: test is not defined`.How can I remove the test runner related import?
![image](https://github.com/user-attachments/assets/d3c2b276-1e81-4dda-a30b-a3f30f4fbd3f)
![image](https://github.com/user-attachments/assets/912e4e2b-0a2d-41ea-bd85-6d07a059ed52)
![Snipaste_2024-12-15_16-42-28](https://github.com/user-attachments/assets/fd6d5ddf-81d6-40af-b30b-096e30a8088f)

**Comment by christian-bromann** [comment]
@codenoobforreal can you share your project?
